### PR TITLE
CRDCDH-1994 Update Access Request to allow requesting based on specific studies

### PR DESCRIPTION
**Pull request by Alejandro-Vega**
### Overview

Removed selecting specific orgs while requesting access and instead allow a user to request based on specific studies.

> [!NOTE]
> The BE does not currently support these changes.

### Change Details (Specifics)

- Remove the organizations autocomplete field and deleted any mention of listOrganizationNames API
- Added required studies select dropdown using listApprovedStudies API
- Updated tests to include dropdown since it is required

### Related Ticket(s)

[CRDCDH-1994](https://tracker.nci.nih.gov/browse/CRDCDH-1994) (Task)
[CRDCDH-1903](https://tracker.nci.nih.gov/browse/CRDCDH-1903) (US)

### CODE REVIEW

File: src/components/AccessRequest/FormDialog.tsx [hunk 1]
```diff
@@ -88,43 +87,47 @@ const FormDialog: FC<Props> = ({ onClose, ...rest }) => {
...
+    }
+  );
 
   const [requestAccess] = useMutation<RequestAccessResp, RequestAccessInput>(REQUEST_ACCESS, {
     context: { clientName: "backend" },
     fetchPolicy: "no-cache",
   });
 
-  const sortedOrgs = useMemo<string[]>(
+  const sortedStudies = useMemo<ApprovedStudy[]>(
```
Comment (amattu2) [inline src/components/AccessRequest/FormDialog.tsx:114]: This works fine, but should we use the API sorting instead?

**Review by amattu2 — APPROVED** [review]
Everything LGTM. I did annotate one suggestion, but I'm also fine with leaving it unchanged.

Approving AS-IS, feel free to merge if you disagree with the suggestion.

File: src/components/AccessRequest/FormDialog.tsx (hunk 1 above)
Comment (Alejandro-Vega) [reply src/components/AccessRequest/FormDialog.tsx]: Agreed, switched to API sorting in [3171a0f](https://github.com/CBIIT/crdc-datahub-ui/pull/553/commits/3171a0f82a5ee4740d524469d8515efa0eb1658c) and [d20a75f](https://github.com/CBIIT/crdc-datahub-ui/pull/553/commits/d20a75f472d61bae655c91b4fb412bd6d19fd29a).

**Review by amattu2 — APPROVED** [review]
LGTM!

**Pull request merged**